tests/dummy.js

```diff
@@ -0,0 +1,9 @@
+const article1 = {
```

Comment:
we already have a file which holds all dummy data, please make use of that

Reply:
Please also seed article data in the db that will be used by other peers as they work on this object